content/docs/lists-and-keys.md

```diff
@@ -109,7 +109,7 @@ const listItems = numbers.map((number) =>
 );
 ```
 
-The best way to pick a key is to use a string that uniquely identifies a list item among its siblings. Most often you would use IDs from your data as keys:
+Ang pinakamahusay na paraan sa pagpili ng key ay ang paggamit ng string na natatanging kumikilala sa list item at naiiba ito sa ibang items. Ang key ay kadalasang ang IDs galing sa iyong data.
```

Comment:
```suggestion
Ang magandang na paraan sa pagpili ng key ay ang paggamit ng string na natatanging kumikilala sa list item at naiiba ito sa ibang items. Ang key ay kadalasang ang IDs galing sa iyong data.
```

Reply:
@elrmndo We'll stick with `pinakamahusay` because the english word used is the `best`. `maganda` is `good` or `better`, it can be `pinakamaganda` but `pinakamahusay` is a proper translation. my 2c let me know.